### ISSUE TITLE
Nightly Builds:  Please Give Overview in Plain English

### ISSUE THREAD

**Issue by PsmithG** [opentoonz/opentoonz#673]
I am sure the developers want ordinary users to test the Nightly Builds - and then give feedback regarding their experiences.

Most OpenToonz users, however, do not speak "Code" - and do not understand the programmer-only  Lingo that is used to describe the changes or improvements contained in each Build.

Many more users will immediately begin to give feedback and bug reports if they know what it is they are testing - and whether they really need to download any particular Nightly Build - based on the Plain English description of what it is and what it fixes or adds.

It is often safer to only test new Builds which contain the particular, sought after features and fixes that one is looking for.

Greg Smith

**Comment by turtletooth** [comment]
Nightly builds include every addition made since the latest release.  They receive the info related to the most recent change in their description, but if multiple changes are made over the course of a day, the description will only have the most recent update.  The process is automated and not something that is really practical to customize for each nightly build.

The best thing to do is report errors as you see them.
